build/package/circleci/orb.yml

```diff
@@ -114,6 +114,7 @@ jobs:
         default: false
     docker:
       - image: launchdarkly/ld-find-code-refs:0.6.0
+        entrypoint: sh
```

Comment:
I spent some time tweaking the circle orb config to try to get it to place nice with the entrypoint, but was not able to do anything that clean. So, I resorted to overriding the entrypoint in the circle orb instead of coming up with a simpler config.

If anyone is familiar with how to do this sort of thing, I'd appreciate feedback on how we could make this better.

Reply:
This seems right to me.